openmetadata-ui/src/main/resources/ui/src/pages/RolesPage/AddRolePage/AddRolePage.tsx

```diff
@@ -18,21 +18,15 @@ import { useHistory } from 'react-router-dom';
 import { addRole, getPolicies } from '../../../axiosAPIs/rolesAPIV1';
 import RichTextEditor from '../../../components/common/rich-text-editor/RichTextEditor';
 import TitleBreadcrumb from '../../../components/common/title-breadcrumb/title-breadcrumb.component';
-import {
-  GlobalSettingOptions,
-  GlobalSettingsMenuCategory,
-} from '../../../constants/globalSettings.constants';
+import { GlobalSettingOptions } from '../../../constants/globalSettings.constants';
 import { Policy } from '../../../generated/entity/policies/policy';
-import { getRoleWithFqnPath, getSettingPath } from '../../../utils/RouterUtils';
+import { getPath, getRoleWithFqnPath } from '../../../utils/RouterUtils';
 import { showErrorToast } from '../../../utils/ToastUtils';
 const { Option } = Select;
 const breadcrumb = [
   {
     name: 'Roles',
-    url: getSettingPath(
-      GlobalSettingsMenuCategory.ACCESS,
-      GlobalSettingOptions.ROLES
-    ),
+    url: getPath(GlobalSettingOptions.ROLES),
```

Comment:
Can we take this to constant and use it here and in other places?
```js
const rolesPath=getPath(GlobalSettingOptions.ROLES)
```
```suggestion
    url: rolesPath,
```